apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -173,52 +200,52 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
 
     // Wizard
 
-    const handleFormSubmit = (values, form, callback) => {
-
-        let identityProvider: IdentityProviderInterface;
+    /**
+     * @param values {object} form values
+     * @param _ form {Form} form Instance
+     * @param __ callback
+     */
+    const handleFormSubmit = (values, _, __) => {
+
+        const FIRST_ENTRY = 0;
+
+        /**
+         * We use a grouped template to keep the sub templates of enterprise
+         * identity provider templates. So, before assigning the form values to the
+         * template first we need to find the correct sub template and deep
+         * clone that object to avoid mutation on file level configuration.
+         */
+        const { idp: identityProvider } = showAsStandaloneIdentityProvider ?
+            template
+            :
+            cloneDeep(template.subTemplates.find(({ id }) => {
+                return id === (selectedProtocol === "saml" ?
+                        IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.SAML :
+                        IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.OIDC
+                );
+            }));
```

Comment:
Shall we reformat this so that `?` and `template` are on the same line? And shall we do the same for `:` as well?